File: fixtures/RH003/after/CalculatorTest.java
import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class CalculatorTest {

    @Disabled("broken")
    @Test
    void addsTwoNumbers() {
        assertEquals(3, Calculator.add(1, 2));
    }
}

File: fixtures/RH003/after/CalculatorTest.kt
import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test
import org.junit.jupiter.api.Assertions.assertEquals

class CalculatorTest {

    @Disabled
    @Test
    fun addsTwoNumbers() {
        assertEquals(3, Calculator.add(1, 2))
    }
}
